autoscale-docker-swarm-container/pom.xml

```diff
@@ -0,0 +1,253 @@
+                                        <args>-v</args>
+                                        <args>--</args>
+                                        <args>/maven/scaler.sh</args>
+                                    </exec>
+                                </entryPoint>
+                                <assembly>
+                                    <mode>tar</mode>
+                                    <inline>
+                                        <fileSets>
+                                            <fileSet>
+                                                <includes>
+                                                    <include>*.sh</include>
+                                                </includes>
+                                                <fileMode>0755</fileMode>
+                                            </fileSet>
+                                            <fileSet>
+                                                <includes>
+                                                    <include>*.yaml</include>
+                                                </includes>
+                                            </fileSet>
+                                            <fileSet>
+                                                <directory>src/main/config</directory>
+                                                <outputDirectory>config</outputDirectory>
+                                            </fileSet>
+                                        </fileSets>
+                                        <dependencySets>
+                                            <dependencySet>
+                                                <useProjectArtifact>false</useProjectArtifact>
+                                                <scope>runtime</scope>
+                                                <excludes>
+                                                    <exclude>com.github.krallin:tini</exclude>
+                                                </excludes>
+                                            </dependencySet>
+                                            <dependencySet>
+                                                <useProjectArtifact>false</useProjectArtifact>
+                                                <fileMode>0755</fileMode>
+                                                <includes>
+                                                    <include>com.github.krallin:tini</include>
+                                                </includes>
+                                            </dependencySet>
+                                        </dependencySets>
+                                    </inline>
+                                </assembly>
+                            </build>
+                            <run>
+                                <extraHosts>
+                                    <host>vagrant-mesos:192.168.56.10</host>
```

Comment:
As the container starts up and is using the default ${docker.host} information, it takes the users env information in the DOCKER_HOST env, mentioned for users who have the name vagrant-mesos in the their DOCKER_HOST variable.  e.g. http://vagrant-mesos:2375.  This enables communication to work in this case.  I didn't want to force a change on users environments over something so simple.